### PR TITLE
Moving general lad sinks config back to protected from public. 

### PR DESCRIPTION
**Pull request by nidhanda**
The portal UI team needs to have AzMonSink config in public. So we had moved the entire sinks config from protected to public. But a better design approach is to only move AzMonSink config in public as the other sinks config can have urls and secrets that are better left in protected.

### CODE REVIEW

File: Diagnostic/diagnostic.py [hunk 1]
```diff
@@ -238,11 +238,10 @@ def encrypt_string(cert, secret):
     global enable_metrics_ext
     ladconfig = configurator._ladCfg()
     sinks = LadUtil.getFeatureWideSinksFromLadCfg(ladconfig, 'performanceCounters')
```
Comment (hestolz) [inline Diagnostic/diagnostic.py:240]: `sinks` no longer being used

File: Diagnostic/diagnostic.py [hunk 2]
```diff
@@ -238,11 +238,10 @@ def encrypt_string(cert, secret):
     global enable_metrics_ext
```
Comment (hestolz) [inline Diagnostic/diagnostic.py:238]: is this still being used

**Review by hestolz — CHANGES_REQUESTED** [review]

File: Diagnostic/diagnostic.py (hunk 2 above)
Comment (nidhanda) [reply Diagnostic/diagnostic.py:238]: Yeah it is used down in the main and start_mdsd methods

File: Diagnostic/diagnostic.py (hunk 1 above)
Comment (nidhanda) [reply Diagnostic/diagnostic.py:240]: Ah true. removing it now

**Review by hestolz — APPROVED** [review]

**Pull request merged**